afl-fuzz.c

```diff
@@ -915,54 +915,50 @@ static inline u8 has_new_bits(u8* virgin_map) {
        that have not been already cleared from the virgin map - since this will
        almost always be the case. */
 
-    if (unlikely(*current) && unlikely(*current & *virgin)) {
+    if (unlikely(*current)) {
+      u8* edg = (u8*)edge;
+      u8* cur = (u8*)current;
+
+#ifdef __x86_64__
+    for (u8 j = 0; j < 8; j++) 
```

Comment:
Whitespace nit: You probably need to fix indentation for everything until `edge++;`. Add two spaces.